plugins/node/opentelemetry-instrumentation-hapi/src/utils.ts

```diff
@@ -94,6 +94,20 @@ export const getRouteMetadata = (
   };
 };
 
+export const getRootSpanMetadata = (
+  route: Hapi.ServerRoute
+): {
+  attributes: SpanAttributes;
+  name: string;
+} => {
+  return {
+    attributes: {
+      [SemanticAttributes.HTTP_ROUTE]: route.path,
+    },
+    name: `${route.method} ${route.path}`,
```

Comment:
can a `route.path` be empty if so then

```suggestion
    name: `${route.method} ${route.path || '/'}`,
```

Reply:
from looking at hapi docs it looks like the path must start with `/` so I don't think this is necessary.
```
path - (required) the absolute path used to match incoming requests (must begin with '/'). Incoming requests are compared to the configured paths based on the server's router configuration. The path can include named parameters enclosed in {} which will be matched against literal values in the request as described in Path parameters.
```
https://github.com/hapijs/hapi/blob/master/API.md#-serverrouteroute

Reply:
make sense, thx